operate/ledger/profiles.py

```diff
@@ -110,6 +110,8 @@
     },
 }
 
+DEFAULT_MECH_MARKETPLACE_PRIORITY_MECH = "0x552cEA7Bc33CbBEb9f1D90c1D11D2C6daefFd053"
```

Comment:
@jmoreira-valory wdyt about this?
```suggestion
PRIORITY_MECH_ADDRESS: t.Dict[Chain, str] = {
    Chain.GNOSIS: "0x552cEA7Bc33CbBEb9f1D90c1D11D2C6daefFd053",
}
```